### PR TITLE
19483 - Staff Payment Restore

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#19483

*Description of changes:*
- Ticket #19483
- Updated App.vue to enable Staff Payment data saved and restored
- Completed validation, checked unit tests have been added for new components and pages

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -700,6 +700,13 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         throw error // go to catch()
       })
 
+      // get Keycloak roles
+      const keycloakRoles = await this.loadKeycloakRoles().catch(error => {
+        console.log('Keycloak roles error =', error) // eslint-disable-line no-console
+        this.accountAuthorizationDialog = true
+        throw error // go to catch()
+      })
+
```
Comment (ArwenQin) [inline src/App.vue:709]: This corrects the issue that Staff payment data is not restored

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-715-eb9xp2a6.web.app

**Review by severinbeauvais — APPROVED** [review]

**Comment by severinbeauvais** [comment]
I re-ran the failed jobs (esp upload to codecov) because they're not supposed to fail any more with @JazzarKarim 's fix yesterday. Hopefully they'll succeed this time.

**Comment by severinbeauvais** [comment]
^^ ignore failed test-coverage test for now

**Comment by JazzarKarim** [comment]
Very weird, it should've worked 😢 I'll have another look.

**Comment by JazzarKarim** [comment]
It worked for the PR you merged 2 hours ago Sev as well. Hmm, weird.

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

**Comment by ArwenQin** [comment]
> test-coverage

 test-coverage still failed.
Is it ok to merge?

**Pull request merged**